README.md

```diff
@@ -1,13 +1,50 @@
-# Slothstagram
+## Slothstagram
 
-# Setting up
+## Setting up
 
-- `git clone` the repo
-- Make virtual enviornment and with within this enviornment for all changes with `mkvirtualenv slothstagram`
-- NOTE: When changing to another project use `deactivate` to exit the virtual enviornment. Use `workon slothstagram` to return to this project's enviornment.
+- Clone this repository
+- Create python virtual environment and make all changes within this environment.
+
+_NOTE: When changing to another project use `deactivate` to exit the virtual environment and `workon slothstagram` to return to this project's environment._
```

Comment:
Can you list out the steps of creating, using and deactivating the virtual environment?
And add a link to the tool we are using to manage virtual environments, VirtualEnvWrapper